tests/test_geoprocessing.py

```diff
@@ -4166,8 +4166,9 @@ def test_convolve_2d_nodata(self):
         target_array = pygeoprocessing.raster_to_numpy_array(target_path)
         target_nodata = pygeoprocessing.get_raster_info(
             target_path)['nodata'][0]
+        self.assertIsNotNone(target_nodata)

```

Comment:
Does it make sense to assert that it is equal to float32 min since that's what the docstring says it should be?

Reply:
Done!